src/lib/executeSourceTransfer.js

```diff
@@ -2,10 +2,17 @@
 
 const log = require('../common').log.create('executeSourceTransfer')
 const validator = require('./validate')
+const moment = require('moment')
+const promiseRetry = require('promise-retry')
+const retryOpts = { // 22 retries over approx. 24 hours
+  retries: 22,
```

Comment:
The retry module allocates some data structure based on this number and, hence, if you set it to Infinity the node process crashes with an out of memory error.